core/src/main/java/org/jdbi/v3/core/statement/StatementContext.java

```diff
@@ -370,7 +405,7 @@ public boolean isConcurrentUpdatable() {
      *
      * @param concurrentUpdatable if the result set should be concurrent updatable.
      */
-    void setConcurrentUpdatable(final boolean concurrentUpdatable) {
+    public void setConcurrentUpdatable(final boolean concurrentUpdatable) {
```

Comment:
These had to be made public to be accessible across core / statement packages. I don't think this configuration for the statement builder really belongs here in `StatementContext`. I wonder if it could live on the StatementBuilder.